src/common/dom-helpers/input-blur.directive.ts

```diff
@@ -0,0 +1,20 @@
+import {Directive, OnInit, Input, Renderer2, ElementRef} from '@angular/core';
```

Comment:
I've named it so because of the selector: that is the hardcoded 'input' in this case:
`ngOnInit() {
 +		this.listen('input', 'blur', this.onBlur);
 +	}`
And before refactoring the code was like this:
`this.inputElement = this.renderer.selectRootElement('input');
...
this.renderer.listen(this.inputElement, 'blur', () => {
 -			setTimeout(this.onBlur, 200);`

To get rid of the 'input' prefix we need to rewrite the code, so the directive selector receives the value from the argument dynamically instead of being hardcoded input.

Reply:
according to this remark, the 'input-blur directive' has been renamed to 'blur directive' in the commit: [blur directive, dom base class: refactoring](https://github.com/qgrid/ng2/pull/39/commits/bb9b43a5374e93ada17ace5c6f028fd5aa7ac651).
The directive has been refactored, so it now receives the input property value through the 'q-grid-on-blur-selector' attribute that has the 'input' value by default